README.md

```diff
@@ -89,25 +89,20 @@ pong_agent = PingPongAgent(0)
 register(container2, ping_agent)
 register(container, pong_agent)
 
-# Start the containers. At this point the TCP-server is created and bound
-# to their addresses
-wait(Threads.@spawn start(container))
-wait(Threads.@spawn start(container2))
-
-# Send the initial message from the ping_agent to initiate the communication
-wait(send_message(ping_agent, "Ping", AgentAddress(aid=pong_agent.aid, address=InetAddr(ip"127.0.0.1", 2939))))
-
-# Wait until some Pings and Pongs has been exchanged
-wait(Threads.@spawn begin
-    while ping_agent.counter < 5
-        sleep(1)
-    end
-end)
-
-# Gracefully shutdown the container and its containing agents
-@sync begin
-    Threads.@spawn shutdown(container)
-    Threads.@spawn shutdown(container2)
+# Start the Mango.jl system. At this point the TCP-server is created and bound
+# to their addresses. After that, the runnable is executed (do ... end). at the 
+# end the container and therefor the TCP server are shut down again. Using this 
+# method it is not possible to forget starting or stopping containers.
+run_mango([container, container2]) do 
```

Comment:
I am not convinced by `run_mango` as the function name since Mango is way more than just the container and this one runs the container.

`run_container`, or even `activate_container`? I think I like `activate` more than `run` for what this does atm.

Reply:
done